app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -252,9 +252,9 @@ public ChatMessage getItem(int index) {
         if (getData() != null && getData().isValid()) {
             if (index == getData().size()) {
                 if (isSusiTyping) {
-                    return new ChatMessage(-404, "", "", false, false, false, "", null, "", 0);
+                    return new ChatMessage(-404, "", "", false, false, false, "", null, "", "");
                 }
-                return new ChatMessage(-405, "", "", false, false, false, "", null, "", 0);
+                return new ChatMessage(-405, "", "", false, false, false, "", null, "", "");
```

Comment:
Not absolutely necessary, but is better to write inline comment mentioning it in such peculiar cases so that it is easier for other contributors to understand the purpose.